### ISSUE TITLE
Feature Request: Combine Uninstall/Install Mods

### ISSUE THREAD

**Issue by Deathstryker** [ClusterM/hakchi2#135]
I think it would be less confusing to combine these two and just select/unselect check boxes for the mods you want to install/uninstall all in one place. Have the program remember what you have checked (installed) and unchecked (uninstalled) the next time you use this feature.

**Comment by abijanfar** [comment]
+1 for this request.

**Comment by sanjay900** [comment]
I think the real issue there is is what happens when you use multiple consoles?

**Comment by Deathstryker** [comment]
Either way,  you still have to remember what is installed on another console due to hakchi not being able to detect what mods are installed so I don't see it being an issue.

**Comment by abijanfar** [comment]
I think you should have a different hakchi folder for each console. This is also recommended by Cluster.

**Comment by sanjay900** [comment]
True, i don't actually have multiple consoles. I just saw in another issue people were doing that.

**Comment by ClusterM** [comment]
We can read installed mods list from NES Mini now (it's already in 2.14) but we can do it while NES Mini powered on normally only. And for installing/uninstalling mods we need to put console into FEL mode. It's confusing too.
Also most mods (near all) can be installed while system is loaded, not while preinit step. But how to determine this mods...
